app/Models/Solo/Score.php

```diff
@@ -127,6 +131,30 @@ public function getMode(): string
         return Beatmap::modeStr($this->ruleset_id);
     }
 
+    /**
+     * Queue the item for score processing
+     *
+     * @param array $dataJson Pre-generated json of the score so it doesn't need to be redundantly generated
+     * @param bool $force By default only newly created scores are queued. Pass true to force queuing the score
+     */
+    public function queueForProcessing(?array $dataJson = null, bool $force = false): void
```

Comment:
Is the nullable or flag being used somewhere?

Reply:
this is a weird function and those are originally so its main function (queuing score for processing) can be accessed regardless of state and existence of pre-generated json...